manifests/2.7.1/opensearch-dashboards-2.7.1.yml

```diff
@@ -10,3 +10,7 @@ components:
   - name: OpenSearch-Dashboards
     repository: https://github.com/opensearch-project/OpenSearch-Dashboards.git
     ref: '2.7'
+  - name: searchRelevanceDashboards
+    repository: https://github.com/opensearch-project/dashboards-search-relevance.git
+    ref: tags/2.7.1.0
```

Comment:
Hi @sejli tags are not created until this version released. For CI stage, we would use the branch for reference here. Could you make sure you have bump 2.7 branch to 2.7.1 and use 2.7 here?

Reply:
Changed in [f450026](https://github.com/opensearch-project/opensearch-build/pull/3494/commits/f450026de4af92149cbe14c59fa98707af90dada)